rpm/packagescanner.go

```diff
@@ -30,21 +26,21 @@ var (
 	_ indexer.PackageScanner   = (*Scanner)(nil)
 )
 
-// Scanner implements the scanner.PackageScanner interface.
+// Scanner implements the [indexer.PackageScanner] interface.
 //
 // This looks for directories that look like rpm databases and examines the
 // files it finds there.
 //
 // The zero value is ready to use.
 type Scanner struct{}
 
-// Name implements scanner.VersionedScanner.
+// Name implements [indexer.VersionedScanner].
 func (*Scanner) Name() string { return pkgName }
 
-// Version implements scanner.VersionedScanner.
+// Version implements [indexer.VersionedScanner].
 func (*Scanner) Version() string { return pkgVersion }
```

Comment:
Just confirming: even though the code is very different, we aren't expecting different results, right?

Reply:
Right.